.tidyrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "indent": 2,
+  "ribbon": 1,
+  "typeArrowPlacement": "first",
+  "unicode": "never",
+  "width": null
```

Comment:
Do you prefer `null` to omitting the key? I feel that that's the more common approach in configuration (null values are not included).

Reply:
I prefer all the options being there. I should do the same for `operators`.